README.md

```diff
@@ -30,93 +24,80 @@ the BMC and the host of the machine(s) you're testing.
 
 ### Preparation ###
 
-**Machine Configuration**
+The target system will need to have an OS that can boot. That OS will
+need to have several things installed on it.
 
-Copy the bvt/op-machines-example.xml file and use its layout (specified
-in bvt/op-machines.xsd) to specify the machines in your test lab.
+A TODO item is to document what that is.
```

Comment:
(anyway, I've fixed up the README with the suggestions, bar git and gcc, as I'd prefer we don't automatically install/compile things)